services/ai-navigator-app/0.1.0/ai-navigator-app.yaml

```diff
@@ -0,0 +1,52 @@
+# TODO: Should these be separate files?
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ai-navigator-app-config
+  namespace: kommander
+data:
+  CHATBOT_URL: "https://chatbot.api.d2iq.com/v1/chat"
+  AUTHORIZER_URL: "https://auth.api.d2iq.com"
+  LICENSE_NAMESPACE: "kommander" # TODO: use kommander namspace, is this hardcoded ok?
```

Comment:
for the namespace, you can use substitution variables like `${releaseNamespace}`

i.e.
https://github.com/mesosphere/kommander-applications/blob/2853c7f030ad633ca23e4f6f02aefdda3cce1555/services/kommander/0.7.0/kommander.yaml#L6

Reply:
Done ✅